src/main/java/jewellery/inventory/mapper/ResourceMapper.java

```diff
@@ -0,0 +1,43 @@
+package jewellery.inventory.mapper;
+
+import jewellery.inventory.dto.request.resource.*;
+import jewellery.inventory.dto.response.resource.*;
+import jewellery.inventory.exception.MappingException;
+import jewellery.inventory.model.resource.*;
+
+public class ResourceMapper {
```

Comment:
I think instead of class we should be using interface in this case because of the mapstruct library. It uses the interface so it can generate ResourceMapperImpl class with the necessary methods. I am not sure why it works with class also 🗡️

Reply:
Did you use maven clean install? The code in the Impl class generates after clean install. This is my UserMapperImpl
![image](https://github.com/JewelleryManagement/jewellery-management-service/assets/4009642/203b1722-55c8-4af1-b44f-953e73a42070)

Reply:
This is because you have empty methods that you let MapStruct library generate for you. In the case of ResourceMapper, it doesn't have anything to generate.